app/templates/main/search-transferring-body.html

```diff
@@ -11,16 +11,16 @@
     {%- set _ = breadcrumbs.update({'search_terms': breadcrumb_values[3]["search_terms"]}) -%}
 {% endif %}
 {% set sorting_list = {
-    "Series reference (A to Z)": "series-asc",
-    "Series reference (Z to A)": "series-desc",
+    "Series reference (A to Z)": "series_id-asc",
```

Comment:
does this change mean sorting is working with opensearch in this PR?

Reply:
was just me tinkering around with it haha